docs/platform-support.rst

```diff
@@ -215,3 +215,33 @@ NVIDIA Network Operator has been validated in the following scenarios:
      - Yes
      - 
 
+
+=============================================
+Supported Precompiled Images for DOCA Drivers
+=============================================
+
+--------
+Overview
+--------
+
+To save startup (compilation) time and operational effort, precompiled DOCA driver images are available for common OS/flavor/kernel/architecture variants.
+
+The container image tag pattern used for common variants is: **driver_ver-container_ver-kernel_ver-flavor-os-arch**. For example: ``24.07-0.6.1.0-6.8.0-49-generic-ubuntu24.04-amd64``
+
+**NOTE:** The images are also packaged as multi-architecture manifests, so manually stating architecture component is optional.
+
+**NOTE:** For the *generic* flavor of Ubuntu, the default Kernel version is used for precompiling (e.g. *6.8.0-31* for Ubuntu 24.04). Whereas for all other flavors, their latest (at time of DOCA packaging/release) Kernel version is used.
+
+----------------------------
+Limitations and Restrictions
+----------------------------
+
+- Support for deploying the driver containers with precompiled drivers is currently limited to hosts with Ubuntu 22.04 or 24.04 (both amd64 and arm64).
+- NVIDIA supports precompiled driver containers for the most recently released DOCA-OFED GA drivers.
```

Comment:
Not sure what this means.
Each Network Operator version comes with a specific DOCA driver version.

Also `OFED` should not be used

Reply:
I don't know about that. @kvadrage (Alex.P) wrote it...

How would you suggest to alter these notes then?

Reply:
I suggest to remove this sentence.
Maybe add link to NGC in the section before.
See available precompiled container images in [NGC](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/mellanox/containers/doca-driver/tags).

I will let @kvadrage comment here.